<ui version="4.0" >
 <class>FormatDialog</class>
 <widget class="QDialog" name="FormatDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Format</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QCheckBox" name="boldCheckBox" >
       <property name="text" >
        <string>Fettgedruckt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="italicCheckBox" >
       <property name="text" >
        <string>Kursiv</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="underlineCheckBox" >
       <property name="text" >
        <string>Unterstrichen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="strikeOutCheckBox" >
       <property name="text" >
        <string>Durchgestrichen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="foregroundColorPushButton" >
       <property name="text" >
        <string>Farbe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backgroundColorPushButton" >
       <property name="text" >
        <string>Hintergrundfarbe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTextBrowser" name="textBrowser" >
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Grüner Mann zur späten Stunde, wollte nicht mal zu dem Hunde.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormatDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>208</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel" >
     <x>261</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormatDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>124</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel" >
     <x>167</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
